### ISSUE TITLE
Error compiling default_page_data.cpp, line too long.

### ISSUE THREAD

**Issue by evoskuil** [libbitcoin/libbitcoin-protocol#211]
VS2017
```
1>..\..\..\..\src\web\default_page_data.cpp(222): error C2026: string too big, trailing characters truncated
```
The default page data should probably not be in protocol, which should be more general purpose. Ideally this would be incorporated into server.

**Comment by thecodefactory** [comment]
Ok, can do.  The idea was to not have to cut a new release of `bs` on HTML/web changes, but I guess it's not ideal in either and makes more sense there.

**Comment by evoskuil** [comment]
There would still be a new release if protocol changed.

**Comment by thecodefactory** [comment]
This issue was addressed by #213
